core/src/sharedTest/resources/visualization/visualization.json

```diff
@@ -314,7 +314,12 @@
   "numberType": "VALUE",
   "aggregationType": "DEFAULT",
   "legend": {
-    "hidden": false
+    "showKey": false,
```

Comment:
We can use this info to verify deserialization in VisualizationShould unit test.